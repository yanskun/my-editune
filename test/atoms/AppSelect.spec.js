import { shallowMount } from "@vue/test-utils"
import Component from "~/components/atoms/AppSelect"

describe("AppSelect", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        value: "",
        placeholder: "placeholder",
        options: [
          { value: "value1", label: "option1" },
          { value: "value2", label: "option2" }
        ]
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
