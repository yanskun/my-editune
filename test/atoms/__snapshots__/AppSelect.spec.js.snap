// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSelect snapshot 1`] = `
<el-select-stub
  autocomplete="off"
  multiplelimit="0"
  placeholder="placeholder"
  popperappendtobody="true"
  value=""
  valuekey="value"
>
  <el-option-stub
    label="option1"
    value="value1"
  />
  <el-option-stub
    label="option2"
    value="value2"
  />
</el-select-stub>
`;
